app/src/main/cpp/skyline/services/account/IProfile.cpp

```diff
@@ -38,4 +52,40 @@ namespace skyline::service::account {
 
         return {};
     }
+
+    Result IProfile::GetImageSize(type::KSession &session, ipc::IpcRequest &request, ipc::IpcResponse &response) {
+        std::vector<char> profileImageIcon = GetProfilePicture();
+        if(profileImageIcon.empty()){
+            response.Push<u32>(profileImageIconDefault.size());
+        } else {
+            response.Push<u32>(profileImageIcon.size());
+        }
+        return {};
```

Comment:
This doesn't match our coding guidelines, I'd suggest giving it a quick read. It's also not a good idea to read a whole file to get its size.
```suggestion
        std::vector<char> profileImageIcon{GetProfilePicture()};
        if (profileImageIcon.empty())
            response.Push<u32>(profileImageIconDefault.size());
        else
            response.Push<u32>(profileImageIcon.size());
        return {};
```